tests/e2e/test_rosacli_idp.go

```diff
@@ -15,6 +15,14 @@ import (
 	ph "github.com/openshift/rosa/tests/utils/profilehandler"
 )
 
+func validateOuptput(textData string, clusterID string, idpName string) {
```

Comment:
As you define the function at the top level. I would suggest using a prominent function name so there is no conflict with the others in the future, e.g., `validateIDPOutput`.

Reply:
I have updated the PR and i do like your suggestion for function naming, i will try in incorporate this in future,
could you please have a look again? Thanks!